tests/test_handler_transform_inverter.py

```diff
@@ -48,6 +49,7 @@ def test_invert(self):
             [
                 LoadImaged(KEYS),
                 AddChanneld(KEYS),
+                Spacingd(KEYS, pixdim=(1.1, 1.01, 0.9), mode=["bilinear", "nearest"], dtype=np.float32),
```

Comment:
quite annoying we have the shape quantization error as I mentioned before. 
the current test case works fine, but if I change the case to:
```suggestion
                Spacingd(KEYS, pixdim=(1.2, 1.01, 0.9), mode=["bilinear", "nearest"], dtype=np.float32),
```
output:
```
F
======================================================================
FAIL: test_invert (__main__.TestTransformInverter)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Documents/MONAI/tests/test_handler_transform_inverter.py", line 99, in test_invert
    self.assertTupleEqual(i.shape, (1, 100, 101, 107))
AssertionError: Tuples differ: torch.Size([1, 101, 101, 107]) != (1, 100, 101, 107)

First differing element 1:
101
100

- torch.Size([1, 101, 101, 107])
+ (1, 100, 101, 107)

----------------------------------------------------------------------
Ran 1 test in 7.454s

```
I'll try to address it...

cc @rijobro @Nic-Ma